Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/RunRunnable.java

```diff
@@ -0,0 +1,53 @@
+package org.dolphinemu.dolphinemu.features.settings.model.view;
+
+import org.dolphinemu.dolphinemu.features.settings.model.AbstractSetting;
+
+public final class RunRunnable extends SettingsItem
+{
+  private final boolean mConfirmRunnable;
+  private final int mAlertText;
+  private final int mToastTextAfterRun;
+  private final Runnable mRunnable;
+
+  public RunRunnable(int titleId, int descriptionId, boolean confirmRunnable, int alertText,
```

Comment:
What do you think about removing the `confirmRunnable` variable and instead making the code check `alertText != 0`, like you've done with `toastTextAfterRun`?

Reply:
Good idea. I used `alertTextID > 0` to stay consistent with the other ViewHolders.